exporter/exporterhelper/common.go

```diff
@@ -119,7 +120,7 @@ func WithQueue(config QueueSettings) Option {
 // This option should be used with the new exporter helpers New[Traces|Metrics|Logs]RequestExporter.
 // Experimental: This API is at the early stage of development and may change without backward compatibility
 // until https://github.com/open-telemetry/opentelemetry-collector/issues/8122 is resolved.
-func WithRequestQueue(cfg exporterqueue.Config, queueFactory exporterqueue.Factory[Request]) Option {
+func WithRequestQueue(cfg exporterqueue.Config, queueFactory exporterqueue.Factory[internal.Request]) Option {
```

Comment:
This fucntion is part of the public API. We cannot change `Request` to `internal.Request` since it'll break all users. We need to find a way to make it work without moving Request to internal

Reply:
Thank you Dmitrii. I ended up working around the API change by re-exporting Request from `exporterhelper`:https://github.com/open-telemetry/opentelemetry-collector/pull/11042
Let me know if you have any concerns